endpoint/endpoint.go

```diff
@@ -0,0 +1,21 @@
+package endpoint
```

Comment:
I secretly wished this package would be the `end` and have a type called `Point`.